news/2024/2024-07-19-osu-lazer-updates-july-18-2024.md

```diff
@@ -0,0 +1,64 @@
+---
+
+Trade offer received — you click the banner, we give you the latest osu!(lazer) updates.
+
+![](/wiki/shared/news/2024-07-19-osu-lazer-updates-july-18-2024/banner.jpg)
+
+It looks like you accepted the trade offer. Well here it is, the latest and greatest osu!(lazer) has to offer, available for you to consume both in text by continuing to read below, or if you are more of a video digest person, you can click on the embed below.
+
+<div align="center">
+    <iframe width="95%" style="aspect-ratio: 16 / 9;" src="https://www.youtube.com/embed/KXBqNfkeZNM" frameborder="0" allowfullscreen></iframe>
+</div>
+
+## Editor changes
+
+- **The gameplay testing hotkeys were implemented, and new hotkeys were added for slider curve types.** Bookmarks and autoplay speed are still not added as part of the testing hotkeys, but now you can toggle autoplay with `Tab`, pause with `Ctrl` + `P`, exit at initial position with `F1`, and press `F2` to exit the gameplay test at current position. As for the slider curve types, you can toggle different types with `Alt` + `1` through `5`.
+- **You can now jump to specific timestamps by pasting them in the bottom bar in the editor.** This is a very small UI change that will mean a lot to mappers, especially when it comes to modding.
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/editor-timestamp.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+- **Beatmap contents can now be edited externally.** Due to the way osu!(lazer) handles beatmaps, you were not able to just open the songs folder and edit whatever you wanted, however after today, that is possible. The sounds of cheering from mapping enthusiasts can be heard around the world.
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/edit-externally.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+- **Exported maps made in osu!(lazer) should no longer be broken if they are imported to osu!(stable).** This is yet another step that's making mapping in osu!(lazer) closer to reality.
+- **Hexagonal and circular options have been added to the snap grid.** Expanding on the snap grid editing change made last month, more options have been added which will make the mapping journey easier for both veteran and newbie mappers.
+- **Various improvements have been made to the editor UI.** Long story short, timelines have been slightly reduced, colours for timing points and kiai's have been changed, and tooltips on hover have been added to make it clear to newer players what each marker and highlighted section in the timeline mean.
+
+<div align="center" class="osu-md__paragraph">
+    <video width="95%" controls>
+        <source src="https://assets.ppy.sh/media/news/editor-tooltip.mp4" type="video/mp4" preload="none">
+    </video>
+</div>
+
+## Other changes
```

Comment:
the new mod customisation overlay seems missing. it's a fairly large ui change

Reply:
er i can write something about that it just felt complicated for me after watching the video

Reply:
done feel free to submit edits for it